docs/how-to/how-to-use-k8s-with-cri-containerd-and-kata.md

```diff
@@ -125,43 +123,35 @@ $ sudo systemctl daemon-reload
   $ sudo -E kubectl get pods
   ```
 
-## Install a Pod Network
+## Configure Pod Network
 
 A pod network plugin is needed to allow pods to communicate with each other.
+The simplest type of CNI is `bridge`.
+But you can find more about CNI plugins from the [Using `kubeadm` to Create a Cluster](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/#instructions) guide,
```

Comment:
s/to Create a Cluster/to create a cluster/

Reply:
Maybe change to the page title of the target URL will be more suitable: Creating a cluster with `kubeadm`